actions/wit/entrypoint.sh

```diff
@@ -5,7 +5,14 @@ set -euxo pipefail
 : ${INPUT_COMMAND:?'The `command` input argument must be set'}
 
 if [[ -n "$INPUT_FORCE_GITHUB_HTTPS" ]]; then
-  git config --global url.'https://github.com/'.insteadOf 'git@github.com:'
+  # Don't echo the token
+  set +x
+  if [[ -n "$INPUT_TOKEN" ]]; then
+    git config --global url."https://x-access-token:${INPUT_TOKEN}@github.com/".insteadOf 'git@github.com:'
```

Comment:
Can you double check that other regular git/wit operations don't print out the access token value?

BTW, nice job on investigating how we can authenticate against private repos like this!

Reply:
In my experiments I didn't see it leak. It looks to me that the git porcelain doesn't see the rewritten URL.

Reply:
After talking to @jackkoenig offline, I realized that maybe we should make this GitHub Action parameter more generic. The underlying authentication method you're using is just [HTTP Basic Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication), and the `x-access-token` is just a special username. Should we replace this with more generic, HTTP basic auth parameters?

Alternatively, should we give this parameter a much more specific name, such as including `GITHUB` in the name itself, so that it's clear that it's only for use with GitHub?

Reply:
I agree on adding a username field, I know Azure uses a similar mechanism. How about:
`http_auth_username` (with a default value of "x-access-token") and `http_auth_token`?

Reply:
LGTM!